apps/site/assets/ts/schedule/components/schedule-finder/TripStop.tsx

```diff
@@ -14,24 +14,39 @@ const formattedDepartureTimes = (
   routeType: number
 ): ReactElement<HTMLElement> => {
   const { schedule, prediction, delay } = departure;
+  const skippedOrCancelled = prediction
+    ? prediction.schedule_relationship === "skipped" ||
+      prediction.schedule_relationship === "cancelled"
+    : null;
+  const predictionOrScheduleTime =
+    prediction && prediction.time ? prediction.time : schedule.time;
 
-  if (routeType === 2) {
-    if (delay && delay >= 300 && prediction && prediction.time) {
-      return (
-        <>
-          <span className="schedule-table__times--delayed schedule-table__times--delayed-future_stop">
-            {schedule.time}
-          </span>
-          <br className="hidden-sm-up" />
-          {prediction.time}
-        </>
-      );
-    }
-
-    return <>{schedule.time}</>;
+  if (
+    routeType === 2 &&
+    delay &&
+    delay >= 300 &&
+    prediction &&
+    prediction.time
+  ) {
+    return (
```

Comment:
Do we want these times to also be crossed out if skipped or canceled? Or maybe just have an additional condition so this branch is never shown if the stop is skipped or canceled? Maybe a question for @ingridpierre?

Reply:
I can try to look at this this weekend during some expected diversions on Rockport, Kingston, and Middleborough -- this will be somewhat hard to test otherwise (we need to observe an upcoming _and_ delayed commuter rail train that will pass some skipped/cancelled stops). 👯‍♀️

Reply:
@ingridpierre What do you think would be the best behavior here?

Personally I think the code is straightforward enough that I'm not so worried about seeing it live with actual data. We can always put together a simple test case to prove the results are what we want.

Reply:
Paraphrasing my convo with Cristen in Slack. This particular case had me thinking we're potentially crossing out the wrong element. Since we cross out arrival times for CR when they are adjusted, not when they are skipped, you're right to flag that this may need a rethink.

I'm now of the mind that it might make more sense instead to cross out the stop name also/instead, and also apply the sr-only skipped stop message to that. It may flow more naturally on a screenreader as well that way. I had chosen to cross out arrivals initially to mirror what was done for the schedules tab, but that pattern doesn't hold here and it's worth trying it out something else.

Cristen has said she'll test it out and we can take a look.